src/bin/command.rs

```diff
@@ -104,6 +106,25 @@ pub(crate) enum Command {
         reward: bool,
     },
 
+    /// Allow a BLS key to stake
+    StakeAllow {
+        /// Address used to perform the operation
+        #[clap(short, long)]
+        addr: Option<Address>,
+
+        /// BLS Public key to allow
+        #[clap(long)]
+        bls: String,
```

Comment:
The `bls` argument doesn't make sense:
1. We're leaking an implementation detail to the user
2. The user doesn't know what bls is
3. The bls public keys are already known by the user as "keys" (see the `export` subcommand)

My suggestion is to rename this argument to `key` and also given the "short" too.